src/dmd/globals.d

```diff
@@ -253,6 +259,7 @@ extern (C++) struct Param
 
     const(char)[] moduleDepsFile;        // filename for deps output
     OutBuffer* moduleDeps;              // contents to be written to deps file
+    MessageStyle messageStyle;          // style of file/line annotations on messages
```

Comment:
Please give this an initialiser so that it is obvious what the default is.

Reply:
msvc renamed to default, does that do it too?

Reply:
initializer given